client/js/jsonedit.js

```diff
@@ -858,8 +926,12 @@ window.addEventListener('mousemove', function(e) {
 window.addEventListener('mouseup', function(e) {
   if(!jeEnabled)
     return;
-  if(e.target == $('#jeText'))
+  if(e.target == $('#jeText') && jeContext != 'macro') {
     jeGetContext();
+    if (jeContext[0] == 'Tree' && jeContext[1] != undefined)
+      jeCommands.find(o => o.id == 'je_openWidgetById').call();
+      jeGetContext();
+  }
```

Comment:
No idea what this change does. But you probably have your reasons. Oh wait, it opens a widget when you click on the tree?

Looks like either the indentation is wrong or it's missing curly brackets though..

Reply:
Right, I've added the brackets. It's odd that jeGetContext() needs to be called twice in the tree view, but I couldn't find any other way to make this work.